### ISSUE TITLE
Create generic input for all the supported quantum chemistry packages 

### ISSUE THREAD

**Issue by felipeZ** [votca/xtp#318]
Currently the input to run a DFT calculation with any of the supported quantum chemistry packages (**Orca**, **NwChem**, **Gaussian** or **XTP**) must be manually specified.
A new general syntax should be provided to represent all the input. This expressions are subsequently translated to the corresponding quantum package syntax.
For example:
```
settings.basis = ubecppol
settings.gw_basis = aux-ubecppol
settings.functional = PBE 
settings.charge = 0
```

**Comment by felipeZ** [comment]
The YAML format seems a better idea:
```yml
settings:
  basis:
    ubecppol
  functional:
    PBE0  
  threads:
    8  
  charge:
    0
  scratch:
    "/tmp"
```

**Comment by felipeZ** [comment]
The new input file for the user should look like:
```xml
<package>
<name>orca</name>
<charge>0</charge>
<spin>1</spin>
<basisset>ubecppol</basisset>
<threads>8</threads>
<functional>PBE0</functional>
</package>

```

**Comment by JensWehner** [comment]
Not the threads that is passed via commandline

**Comment by JensWehner** [comment]
What a qmpackage has to have

Functional (gaussian has a different definition of b3lyp)
Basisset (we do not use the basis set that comes with the qmpackage instead we  write them out using the votca data)
Aux Basisset ( as above)
Pseudopotential ( as above)
Spin/charge
Tightness of convergence
Polarisation*
Geometry optimisation*
External charges*

 * does not have to have an options keyword but must be settable via the api

**Comment by felipeZ** [comment]
If we are using a general input a `name` field is also mandatory to identify the target qmpackage

**Comment by JensWehner** [comment]
yes

**Comment by JoshuaSBrown** [comment]
So I have a question about this, since it is doing stuff with writing files, is it going to be using any of the file stuff in csg, or is xtp simply going to be handling this stuff internally?

**Comment by felipeZ** [comment]
Hi @JoshuaSBrown, the idea is that XTP has this settings object that knows how to handle the properties of a given quantum package. The goal is just being able to have generic representation for all the `qmpackage` inputs but I am not changing the file writes, just how the `Property` object is manipulated

**Comment by JoshuaSBrown** [comment]
I see thanks for clarifying.

**Comment by felipeZ** [comment]
The new interface is ready but there are no tests for the input generation of any package

**Comment by felipeZ** [comment]
Since the Gaussian and NWChem interfaces are not used they should be remove

**Comment by felipeZ** [comment]
solved by #420 and #445
